mep/books/views.py

```diff
@@ -25,57 +24,3 @@ def get_queryset(self):
             Q(mep_id__icontains=self.q) |
             Q(notes__icontains=self.q)
         ).order_by('title')    # meaningful default sort?
-
-
-
-class ItemCSV(ListView):
-    '''Export of item details as CSV download.'''
-    # NOTE: adapted from PPA: could be extracted as a view mixin for reuse
-    model = Item
-    # order by id for now, for simplicity
-    ordering = 'id'
-    header_row = ['Admin link', 'Database ID', 'Title', 'Year', 'URI',
-                  'Authors', 'MEP ID', 'Notes']
-    # NOTE: omitting editors and translators because it's slow enough
-    # already with just authors
-
-    def get_csv_filename(self):
-        return 'mep-items-%s.csv' % now().strftime('%Y%m%dT%H:%M:%S')
```

Comment:
Can update the filenames for the new person and item downloads to preserve the timestamp, so subsequent downloads are unique?